borg/_hashindex.c

```diff
@@ -47,11 +47,13 @@ typedef struct {
     void *buckets;
     int num_entries;
     int num_buckets;
+    int num_empty;
```

Comment:
it seemed to need less code to account empty buckets rather than tombstones, so i did that.